tests/pytest.ini

```diff
@@ -0,0 +1,5 @@
+# -- FILE: pytest.ini (or tox.ini)
+[pytest]
+DJANGO_SETTINGS_MODULE = test_settings
```

Comment:
Maybe it's a good idea to link it to a new `jobs.settings.testing` file

Reply:
Great!